test/utils/pdrs.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+import test from 'tape';
+import {tableRow} from '../../app/scripts/utils/table-config/pdrs.js';
+import {getProgress} from '../../app/scripts/utils/table-config/pdr-progress.js';
+const d =
+  {
+    'pdrName': 'test-4.PDR',
+    'collectionId': 'MOD09GQ___006',
+    'status': 'running',
+    'provider': 's3_provider',
+    'progress': 0,
+    'execution': 'execuation-arn-7a2136d9-e07a-4d94-9080-aef7206e8192',
+    'PANSent': false,
+    'PANmessage': 'N/A',
+    'stats': {
+      'processing': 0,
+      'completed': 1,
+      'failed': 3,
+      'total': 4
+    },
+    'createdAt': 1519415184827,
+    'timestamp': 1519415185353,
+    'duration': 0.526
+  };
+
+test('test pdr-progress.js getProgress', function (t) {
+  const result = getProgress(d);
+  t.equal(result.percentCompleted, 25);
+  t.equal(result.percentFailed, 75);
+  t.equal(result.granulesCompleted, '4/4');
+  t.end();
+});
+
+test('test pdrs.js tableRow', function (t) {
```

Comment:
Could this test be simplified to:

```javascript
t.equal(col[4](d), 4);
t.equal(col[5](d), d.stats.processing);
t.equal(col[6](d), d.stats.failed);
t.equal(col[7](d), d.stats.completed);
t.end();
```

Reply:
yes. less code, less error.  Why didn't I think this way